openedx/adg/lms/webinars/admin.py

```diff
@@ -148,6 +163,13 @@ class WebinarRegistrationAdmin(admin.ModelAdmin):
     Model admin for webinar registration
     """
 
+    def get_form(self, request, obj=None, change=False, **kwargs):
+        form = super(WebinarRegistrationAdmin, self).get_form(request, obj, **kwargs)
+        form.base_fields['webinar'] = forms.ModelChoiceField(
+            queryset=Webinar.objects.filter(is_cancelled=False, start_time__gte=now())
+        )
```

Comment:
What purpose does it serve?

Reply:
It filters webinar's field to display only upcoming webinars when admin want to register some user in webinars

Reply:
I guess we should not allow registration from admin. We should register this model as read only model

Reply:
I agree with Ahmed. We can override `has_add_permission`, `has_delete_permission`, and `has_change_permission` to return False in WebinarRegistration Admin. There is no such requirement to allow the ADG admin such privileges.

Reply:
Yes, we remove it after pulling latest commits.

Reply:
You do have those changes @muak50. Just remove this change.